metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -1163,10 +1104,9 @@ func (e *enricher) Enrich(events []mapstr.M) {
 				}
 
 				// don't apply pod metadata to module level
-				k8sMeta = k8sMeta.Clone()
 				delete(k8sMeta, "pod")
 			}
-			ecsMeta := meta.Clone()
+			ecsMeta := meta
```

Comment:
nit: not required since the Clone is now handled by the e.getMetadata(...)

Reply:
This is just renaming the variable for clarity. Maps are reference types in Go, so this really just copies a pointer.

Reply:
I know that maps are built-in references ty 🙂 I have to admit that this is the first time I witness a new variable assignment in the same scope for clarity reasons; however, I have seen comments added or renaming the variable at the creation time, for clarity reasons

Reply:
The real meaning of this series of statements is "`ecsMeta` is `meta` without the `kubernetes` key". It was clearer when we were actually cloning `meta`, but we don't need to do that anymore. I can change it to only use `meta` if you think this is confusing.